build.gradle

```
@@ -66,8 +66,7 @@ dependencies {
     implementation 'com.gradle:gradle-enterprise-gradle-plugin:3.12.3'
     implementation 'org.nosphere.gradle.github:gradle-github-actions-plugin:1.3.2'
     implementation 'com.gradle:common-custom-user-data-gradle-plugin:1.8.2'
-    implementation 'org.gradle:test-retry-gradle-plugin:1.4.1'
-    implementation 'me.champeau.gradle:japicmp-gradle-plugin:0.4.1'
+    implementation 'me.champeaugit .gradle:japicmp-gradle-plugin:0.4.1'
```

Mind the gap between the train and the platform!

```suggestion
    implementation 'me.champeaugit.gradle:japicmp-gradle-plugin:0.4.1'
```

---

wow, how did this happen? Probably went to fast switching between windows :)